### PR TITLE
Add specs and refactor ArticleDecorator#authors

### PR DESCRIPTION
**Pull request by twinn**
Sorry my rspec kinda sucks :(

### CODE REVIEW

File: spec/decorators/article_decorator_spec.rb [hunk 1]
```diff
@@ -1,4 +1,56 @@
 require "rails_helper"
 
 RSpec.describe ArticleDecorator do
+  describe "#authors" do
+		subject { described_class.new(article) }
```
Comment (olivierlacan) [inline spec/decorators/article_decorator_spec.rb]: Those are some crazy indents there @twinn :stuck_out_tongue:

**Comment by olivierlacan** [comment]
How dare you be sorry for contributing specs. :-p

**Comment by olivierlacan** [comment]
I think I'm good to merge as long as you follow the two-space indent "convention" on the project. :-)

**Comment by twinn** [comment]
@olivierlacan not sure what happened to my vim config :(, should be good now.

**Comment by olivierlacan** [comment]
@twinn As long as you don't commit as Gary, I'm okay with it. :stuck_out_tongue:

File: spec/decorators/article_decorator_spec.rb [hunk 2]
```diff
@@ -1,4 +1,56 @@
 require "rails_helper"
 
 RSpec.describe ArticleDecorator do
+  describe "#authors" do
+    subject { described_class.new(article) }
```
Comment (olivierlacan) [inline spec/decorators/article_decorator_spec.rb]: Can you also move the subject after the `let`s. I very much follow Given, When, Then for basic structure on Orientation.

Comment (twinn) [inline spec/decorators/article_decorator_spec.rb]: 👍 done.

**Comment by olivierlacan** [comment]
Thanks @twinn!

**Pull request merged**